aptiche/lib/widgets/HomeScreen/drawer.dart

```diff
@@ -10,7 +11,9 @@ import 'package:aptiche/views/drawer/privacy_policy.dart';
 import 'package:aptiche/views/drawer/profile_page.dart';
 import 'package:aptiche/views/splashscreen/user_controller.dart';
 import 'package:aptiche/widgets/HomeScreen/drawer_list_tile.dart';
+import 'package:day_night_switcher/day_night_switcher.dart';
 import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/cupertino.dart';
```

Comment:
Is this really required? Or just material import can do the job?

Reply:
No ,vs code auto imported it ,I will remove it in next pull request.

Reply:
No ,vs code auto imported it ,I will remove it in next pull request.